src/main/kotlin/de/cyface/collector/configuration/Configuration.kt

```diff
@@ -136,6 +177,13 @@ data class Configuration(
             }
         }
 
+        // TODO [2022-12-14] Klemens Muthmann: This code is ported from the previous implementation.
+        // Armin Schnabel has added the original code, but I do not really understand why it is necessary.
+        // The documentation should be improved when I understand why it is necessary to add a * at the end of the
+        // endpoint.
+        /**
+         * Provide the HTTP endpoint configured for this data collector.
+         */
```

Comment:
I think this was added [here](https://github.com/cyface-de/data-collector/commit/d10550908e50fc2b161ccba8d5178d33d71257df) by you, it seems like this was a requirement in the upgraded Vert.X version?

Reply:
As we use `httpEndpoint() { return "api/v3/*" }` in `mainRouter.route(config.getEndpoint()).subRouter(apiRouter);` I guess you have this requirement from [here](https://vertx.io/docs/vertx-web/java/#_routing_by_paths_that_begin_with_something) in the Vert.X documentation.

I.e. we route everything at `api/v3/*****` to the apiV3Router which handles sub-routes.

And everything else by the failure handler:
`mainRouter.route("/*").last().handler(new de.cyface.api.FailureHandler());`